Makefile.frag

```diff
@@ -46,19 +46,17 @@ ext/awscrt.lo: ext/awscrt.c
 
 ext/awscrt.c: build/aws-crt-ffi-static/libaws-crt-ffi.a ext/api.h ext/awscrt_arginfo.h
 
-ext/awscrt_arginfo.h: ext/awscrt.stub.php gen_stub.php
+ext/awscrt_arginfo.h: awscrt.stub.php gen_stub.php
 ifeq ($(GENERATE_STUBS),1)
+	# install awscrt.stub.php to ext/
+	cp -v awscrt.stub.php ext/awscrt.stub.php
```

Comment:
Why? We moved this to the main folder instead of ext/ and making a copy here.

Reply:
PECL removes .php from subdirectory, but not from the root directory. So, move it outside the ext/ is the easiest way.

Reply:
Why do we need this in ext/? I am just confused about why do we need this file at two places.

Reply:
```
        # generate awscrt_arginfo.h
	php gen_stub.php --minimal-arginfo ext/awscrt.stub.php
```

This part will generate the awscrt_arginfo.h into the same folder as `awscrt.stub.php`.
The `gen_sub.php` is from the PHP build scripts, I didn't dig deep into it. We need `awscrt_arginfo.h` in ext/, so I just copy it around here.